Sources/SIL/SIL.swift

```diff
@@ -279,16 +269,45 @@ public enum Instruction {
     // Used as a temporary workaround in parser
     case unknown(_ name: String)
 
-    // https://github.com/apple/swift/blob/master/docs/SIL.rst#unreachable
-    // unreachable
-    case unreachable
-
     // https://github.com/apple/swift/blob/master/docs/SIL.rst#witness-method
     // witness_method $Self, #Comparable."<="!1 : <Self where Self : Comparable> (Self.Type) -> (Self, Self) -> Bool : $@convention(witness_method: Comparable) <τ_0_0 where τ_0_0 : Comparable> (@in_guaranteed τ_0_0, @in_guaranteed τ_0_0, @thick τ_0_0.Type) -> Bool
     // TODO(#28): Figure out witness_method.
     case witnessMethod(_ archeType: Type, _ declRef: DeclRef, _ declType: Type, _ type: Type)
 }
 
+// https://github.com/apple/swift/blob/master/docs/SIL.rst#terminators
+public enum Terminator {
```

Comment:
Is `Terminator` currently `CustomStringConvertible`? I think it should be.

Reply:
Good point, will make it so.